### PR TITLE
Fix ci failing

### LINKED ISSUE #94: CI failing
the CI is failing with the most recent PR

https://github.com/ploomber/soopervisor/runs/7131896045?check_suite_focus=true

### PR DESCRIPTION
**Pull request by grnnja**
## Issue ticket number and link
Closes #94 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added thorough tests (when necessary).
- [x] I have added the right documentation (when needed). Product update? If yes, write one line about this update.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
 *.sublime-project
 *.sublime-workspace
-*.txt
```
Comment (idomic) [inline .gitignore:3]: Why is this removed?

Comment (idomic) [reply .gitignore:3]: Same for the others, I thought those files are part of the build and should be ignored?

**Review by idomic — CHANGES_REQUESTED** [review]
small comment on the .gitignore

File: .gitignore (hunk 1 above)
Comment (grnnja) [reply .gitignore:3]: It's mentioned in the commit message, but basically Neelasha added a lib folder and requirements.txt files which for some reason don't get copied into the container that the tests run in in github actions, causing the tests to fail
